Blammo-wai/Blammo-wai.cabal

```diff
@@ -0,0 +1,92 @@
+cabal-version: 1.18
+
+-- This file has been generated from package.yaml by hpack version 0.36.0.
+--
+-- see: https://github.com/sol/hpack
+
+name:           Blammo-wai
+version:        1.2.1.0
```

Comment:
What's the thinking in starting the new package at this version?

Reply:
A copy/paste mistake :)